GREEDY/Services/ReceiptService.cs

```diff
@@ -10,6 +10,7 @@ public class ReceiptService : IReceiptService
         private readonly IImageFormating _imageFormating;
         private readonly IReceiptCreating _receiptCreating;
         private readonly IDataConverter _dataConverter;
+        private static AutoCorrect AutoCorrect => new AutoCorrect();
```

Comment:
This is the same as writing private static AutoCorrect AutoCorrect {get {return new AutoCorrect();}
A new instance of will be created each time you reference this property.